scripts/MongoAuthCert.pem

```diff
@@ -0,0 +1,60 @@
+Bag Attributes
+    localKeyID: 49 D8 60 F8 16 9B 8E A9 16 95 24 B5 84 71 C5 5C E7 E8 89 D2 
```

Comment:
this is a self-signed cert right?

Reply:
Yes. This cert is used by the data nodes and the config nodes. I added the capability to generate a self signed cert for the mongos node on the fly to get rid of the IgnoreInvalidHostnames property